systers_portal/templates/403.html

```diff
@@ -9,9 +9,17 @@
     <h1>Access Denied</h1>
     <p>Unfortunately, you don't have permission to access this page.</p>
     <p>
-    	<a class="btn btn-lg btn-success" href="#" role="button" style="margin-top: 80px" onclick="history.go(-1); return false;">Back to previous page</a>
+    	<a data-toggle="tooltip" class="btn btn-lg btn-success" href="#" role="button" style="margin-top: 80px" onclick="history.go(-1); return false;">Back to previous page</a>
```

Comment:
We don't need ```href``` and ```onclick``` now. Once you remove this we are good to go with this one. Also squash your commits. Say *Show previous page link when hover on back button on error pages*. If you can come up with a better commit message for your change please go ahead.

Thanks for the work!